.teamcity/performance-test-durations.json

```diff
@@ -1,548 +1,526 @@
 [ {
-  "scenario" : "org.gradle.performance.regression.android.RealLifeAndroidBuildPerformanceTest.clean assembleDebug with clean transforms cache",
+  "scenario" : "org.gradle.performance.regression.android.AndroidIncrementalExecutionPerformanceTest.abi change",
```

Comment:
I don't understand all the changes in this file. Why did all the durations change (mostly it's a significant increase)? Why did so many things move around?

Reply:
You don't need to understand the changes in the file. I re-created the file from the current timings on CI. That is why things moved around. That is the first commit.